sage-ios/SAGE/SAGE/Checkin.swift

```diff
@@ -47,9 +48,9 @@ class Checkin: NSObject {
                 let schoolDictionary = value as! [String: AnyObject]
                 self.school = School(propertyDictionary: schoolDictionary)
             case CheckinConstants.kVerified:
-                self.verified = value as? Bool
+                self.verified = value as! Bool
             case CheckinConstants.kId:
-                self.id = value as? Int
+                self.id = value as! Int
             case CheckinConstants.kComment:
```

Comment:
same thing, check if it can be cast as an int, and if it is, assign to self.verified -- i need to change these for schools and user too